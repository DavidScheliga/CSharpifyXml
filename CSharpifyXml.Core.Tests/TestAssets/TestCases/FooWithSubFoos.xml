<?xml version="1.0" encoding="utf-8"?>
<Foo>
    <Bar>Bar</Bar>
    <SubFoos>
        <SubFoo>
            <Bar>SubBar</Bar>
        </SubFoo>
        <SubFoo>
            <Bar>SubBar</Bar>
        </SubFoo>
    </SubFoos>
</Foo>