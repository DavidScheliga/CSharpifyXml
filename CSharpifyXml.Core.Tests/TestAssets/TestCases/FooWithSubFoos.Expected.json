{
  "Descriptors": {
    "Foo": {
      "ElementName": "Foo",
      "AttributeNames": [],
      "Children": [
        {
          "ElementName": "Bar",
          "HasChildren": false,
          "HasAttributes": false
        },
        {
          "ElementName": "SubFoos",
          "HasChildren": true,
          "HasAttributes": false
        }
      ]
    },
    "SubFoos": {
      "ElementName": "SubFoos",
      "AttributeNames": [],
      "Children": [
        {
          "ElementName": "SubFoo",
          "Count": "2",
          "HasChildren": false,
          "HasAttributes": false
        }
      ]
    },
    "SubFoo": {
      "ElementName": "SubFoo",
      "AttributeNames": [],
      "Children": [
        {
          "ElementName": "Bar",
          "HasChildren": false,
          "HasAttributes": false
        }
      ]
    }
  }
}
